source/features/hide-empty-split-diff-side.tsx

```diff
@@ -0,0 +1,35 @@
+import './hide-empty-split-diff-side.css';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function isUnifiedDiff(): boolean {
+	return select.exists([
+		'[value="unified"][checked]', // Form in PR
+		'.table-of-contents .selected[href*="diff=unified"]' // Link in single commit
+	]);
+}
+
+function init(): void {
+	for (const diffTable of select.all('.js-diff-table')) {
+		for (const side of ['left', 'right']) {
+			if (!select.exists(`[data-split-side="${side}"]:is(.blob-code-addition, .blob-code-deletion)`, diffTable)) {
```

Comment:
Instead of this inner loop and condition, what if you select the data-split-side element and access the side? It doesn’t make sense to query them both specifically

Reply:
But if I select all the `<td>` elements bearing `[data-split-side]`, wouldn't I need to loop over them anyway to check every value of the attribute? :thinking:

Reply:
You're right! I was thinking this piece of code was doing something else.